pisa/stages/data/super_simple_pipeline.cfg

```diff
@@ -32,7 +32,9 @@ input_specs = None
 calc_specs = None
 output_specs = super_simple_binning
 
-
+# Arguments to turn pseudo weight and mean adjustment on and off
+with_pseudo_weight = False
+with_mean_adjust = False
 
 [pipeline]
 
```

Comment:
We haven't had cfgs inside the code.....but maybe it's a good idea....up for debate!

Reply:
do you mean that the .cfg file should be placed somewhere else?

Reply:
right now we have those under `pisa_examples`